tests/AntDesign.Tests/DatePicker/DatePickerTests.razor

```diff
@@ -103,6 +103,53 @@
 		//Assert
 		input.GetAttribute("value").Should().Be(expected);
 	}
+    
+    [Theory]
+    [InlineData("11", "11/")]
+    [InlineData("111", "11/1")]
+    [InlineData("1111", "11/11/")]
+    [InlineData("11111111", "11/11/1111")]
+    [InlineData("111111111", "11/11/1111")]
+    [InlineData("11/11/1111", "11/11/1111")]
+    [InlineData("11/11/11111", "11/11/1111")]
+    public void Mask_is_applied(string inputValue, string expectedValue)
```

Comment:
Please consider the boundary like ("112","11/02/") that the month never more then 20 so change it to 02.

Reply:
Ready. Also for minutes, days, seconds and hours.